sormas-api/src/main/java/de/symeda/sormas/api/person/PersonFacade.java

```diff
@@ -57,6 +57,8 @@
 	 */
 	List<PersonNameDto> getMatchingNameDtos(UserReferenceDto user, PersonSimilarityCriteria criteria);
 
+	boolean checkMatchingNameDtos(UserReferenceDto userRef, PersonSimilarityCriteria criteria);
```

Comment:
If I don't misunderstand anything, this method is only used in the import to determine whether there are any duplicate results; however, it still uses the same method in PersonService as getMatchingNameDtos, only with a limit; what I'm wondering is whether the limit actually makes a difference? Because the query still needs to do all the joins and that kind of stuff. Wouldn't it maybe be better to already use getMatchingNameDtos at that point and then store the result in the CaseImportEntities so that we don't have to do the same query again in the PersonSelectionGrid?

Reply:
the method "checkMatchingNameDtos" was renamed to "checkMatchingNameInDatabase"